clients/src/main/java/org/apache/kafka/clients/admin/MemberDescription.java

```diff
@@ -131,13 +140,30 @@ public Optional<MemberAssignment> targetAssignment() {
         return targetAssignment;
     }
 
+    /**
+     * The epoch of the group member.
+     */
+    public Optional<Integer> memberEpoch() {
+        return memberEpoch;
+    }
+
+    /**
+     * The flag indicating whether a member is classic.
```

Comment:
nit: How about `The flag indicating whether a member with a consumer group uses the classic protocol. It indicates that the member was perhaps not upgraded`?